platform/qt/src/qmapboxgl.cpp

```diff
@@ -925,12 +1088,31 @@ QMargins QMapboxGL::margins() const
     );
 }
 
-void QMapboxGL::addSource(const QString &sourceID, const QVariantMap &params)
+/*!
+    Adds a data source \a id to the map as specified by \l
+    {https://www.mapbox.com/mapbox-gl-style-spec/#root-sources}{the vector tiles specification} with
+    \a params.
+
+    A data source starting with \b ":" stands for a file tracked by the Qt resource system. In this
```

Comment:
Hmmm, I'm revising this. This is actually the behavior on the QML API, not the low level. My bad.